EpamWebSiteTestingFramework/Epam.TestAutomation.Core/ScreenShotMaker/ScreenshotMaker.cs

```diff
@@ -0,0 +1,35 @@
+using Epam.TestAutomation.Core.DriverCreator;
+using Epam.TestAutomation.Utilities.Logger;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Core.ScreenShotMaker;
+
+public static class ScreenshotMaker
+{
+    private static void TakeScreenshot(IWebDriver driver, string methodName, string folderPath)
+    {
+        if (!Directory.Exists(folderPath))
+        {
+            Directory.CreateDirectory(folderPath);
+        }
+
+        var screenShooter = driver as ITakesScreenshot;
+        var screenShot = screenShooter?.GetScreenshot();
+        screenShot?.SaveAsFile($"{folderPath}/{DateTime.Now:MM/dd/yyyy, HH:mm:ss}_{methodName}.jpeg",
```

Comment:
It is not allowed to set DateTime with format MM/dd/yyyy. Please chhange the "/" to ".".